README.md

```diff
@@ -149,7 +151,9 @@ positional arguments:
 optional arguments:
   -h, --help            show this help message and exit
   --output OUTPUT, -o OUTPUT
```

Comment:
If the output has changed to just file, rather than file or directory, then should the arg info be updated to `--output FILE, -o FILE` to make it clear?
```suggestion
  --output FILE, -o FILE
```

Reply:
Good suggestion, I've added the required `metavar` arguments